HTML/createQuiz.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="../CSS/createQuiz.css">
+
+    
+    <title>Game Quiz</title>
+
+</head>
+<body>
+
+    <nav class="navbar navbar-dark bg-custom "> 
+        <div class="navbar-nav mr-auto">
+            <form action="dashboard.html" method="GET">
+                <button class="btn btn-dark"><i class="bi bi-arrow-left"></i></button>
+            </form>
+        </div>
+        <div class="navbar-nav header">
+            <h1>Create Quiz</h1>
+        </div>
+        <div class="navbar-nav ml-auto">
+            <form action="dashboard.html" method="GET">
```

Comment:
Same question as the above: why use "dashboard.html" for action?

Reply:
dashboard.html is where all quizzes are stored and can be viewed, so it's kind of like a homepage. I put those actions there so that when you press the back button in the navbar you are redirected to the dashboard and when you want to delete your quiz it will delete the quiz and then redirected to the dashboard.

Reply:
Also I forgot to say but it is supposed to be from the dashboard.html where you can get to createQuiz.html which is why I thought it would be okay to redirect back to dashboard after the user was done with the quiz creation and editing.

Reply:
It looks like "dashboard.html" is just a placeholder, since this page is not existing yet. Moreover, it would most likely become a nodejs script or RESTFul API eventually, instead of a html page.

Reply:
I would suggest to keep it empty until the later time to update it.